pkg/extensions/search/README.md

```diff
@@ -0,0 +1 @@
+search.md
```

Comment:
Add a little more text here

`
# `search` extension

Details about the APIs are specified [here](./search.md)

`

Or even remove this README.md entirely, this is a new file which was added right?

Reply:
It's a symlink.